easypost/report.py

```diff
@@ -8,7 +8,22 @@ class Report(AllResource, CreateResource):
     def create(cls, api_key=None, **params):
         """Create a report."""
         requestor = Requestor(api_key)
-        url = "%s/%s" % (cls.class_url(), params["type"])
+        class_url = cls.class_url()
```

Comment:
Assigning intermediate variables is good in two scenarios:

1. When the value you are assigning to a variable isn't immediately clear what it's for/doing
2. When that same value will get used multiple times

Since 1 isn't true as the name of the variable is directly in the name of the value and 2 isn't true because it's only used once, we can remove this intermediate variable and clean this code up a bit by simply calling `cls.class_url()` directly in the `url` variable below.